Makefile

```diff
@@ -1,20 +1,22 @@
 include ./make/*.mk
 
 ORG ?= integreatly
-NAMESPACE_PREFIX=redhat-rhmi-
+NAMESPACE_PREFIX ?= redhat-rhmi-
 NAMESPACE=$(NAMESPACE_PREFIX)operator
 PROJECT=integreatly-operator
+RHOAM_PROJECT=managed-api-service
 REG=quay.io
 SHELL=/bin/bash
-TAG ?= 2.7.0
+RHMI_TAG ?= 2.7.0
```

Comment:
Same with the tags?

Reply:
The tags can be moved but the naming would want to be different. The prepare-release.sh script changes the tag values when making a new release.